README.md

```diff
@@ -7,19 +7,11 @@ To check out the source code for FLCL,
 ```bash
     git clone https://github.com/kokkos/kokkos-fortran-interop.git
 ```
-## Branches
-Consider `master` to be the stable branch. The branch `develop` will contain integrated
-features as they move towards release. Then, feature branches as necessary.
 
 # Requirements
 - Compiler suite with both Fortran (F08) and C++ (C++11) support.
 - A build of [Kokkos](https://github.com/kokkos/kokkos) 3.x built with the compiler suite above.
```

Comment:
What about that line?

Reply:
I was planning to remove that line once I require Kokkos 4 in the CMake. I can already remove it, if you prefer.

Reply:
up to you